doc/nrf/ug_thread_tools.rst

```diff
@@ -310,23 +311,13 @@ When working with samples that support wpantund, complete the following steps to
 
         wpantund -I *network_interface_name* -s *ncp_uart_device* -b *baud_rate*
 
-     For example::
+     For example (UART transport)::
 
         sudo wpantund -I leader_if -s /dev/ttyACM0 -b 1000000
 
-   Radio co-processor (RCP)
```

Comment:
So we cannot do this for RCP anymore?

Reply:
That's correct. ot-ncp application, which we were using for connecting to RCP in wpantund, has been removed and is not evailable any more.